### DISCUSSION TITLE
Disconnected from Manager on fresh install of 2.3.30

### DISCUSSION THREAD

**Discussion by danielbidwell** [Security-Onion-Solutions/securityonion#3376] [Unsupported Versions]
I installed ubuntu 18.04 on an AWS instance and installed security onion 2.3.30.  The install seemed to go fine.  Rebooted after install.  The web now shows a "Disconnected from Manager" at the top.  It thinks I am logged in.  clicking on "settings" show a red bar across the top with "Network error".  Trying to logout just times out.

Where do I start to diagnose this?

**Comment by weslambert** [comment]
Are there any clues present in Chrome Dev Tools, or in the logs in `/opt/so/log/soc/` or `/opt/so/log/nginx`?

**Reply by danielbidwell** [reply]
It looks like a logout and log back in usually fixes the problem.
